test/e2e/util_test.go

```diff
@@ -36,6 +36,7 @@ import (
 	k8scontrollerclient "sigs.k8s.io/controller-runtime/pkg/client"
 
 	gtypes "github.com/onsi/gomega/types"
+	v1 "github.com/operator-framework/api/pkg/operators/v1"
 	"github.com/operator-framework/api/pkg/operators/v1alpha1"
 	operatorsv1alpha1 "github.com/operator-framework/api/pkg/operators/v1alpha1"
```

Comment:
Feel free to complete ignore this: v1 -> operatorsv1? It also looks like there's duplicate v1alpha1 imports here too.